### PR TITLE
Chapter1 solutions added

### PR DESCRIPTION
**Pull request by wookenny**
### Solutions for Chapter 1

cc @vrom911 @chshersh

### CODE REVIEW

File: src/Chapter1.hs [hunk 1]
```diff
@@ -477,8 +478,8 @@ Implement a function that returns the last digit of a given number.
   results. Or you can try to guess the function name, search for it and check
   whether it works for you!
 -}
--- DON'T FORGET TO SPECIFY THE TYPE IN HERE
-lastDigit n = error "lastDigit: Not implemented!"
+lastDigit :: Int -> Int
+lastDigit n = (abs n) `mod` 10
```
Comment (vrom911) [inline src/Chapter1.hs:482]: Nice work on using infix form of calling functions! We don't explain this syntax in our course to make it short. But well done :+1: 

Also, you can remove `()`, they are redundant in this case :slightly_smiling_face: 
```suggestion
lastDigit n = abs n `mod` 10
```

File: src/Chapter1.hs [hunk 2]
```diff
@@ -556,8 +559,17 @@ True
...
+isVowel :: Char -> Bool
+isVowel c 
+    | (toLower c) == 'a' = True
+    | (toLower c) == 'e' = True
+    | (toLower c) == 'i' = True
+    | (toLower c) == 'o' = True
+    | (toLower c) == 'u' = True
+    | (toLower c) == 'ä' = True -- German Umlauts included
+    | (toLower c) == 'ö' = True
+    | (toLower c) == 'ü' = True
```
Comment (vrom911) [inline src/Chapter1.hs:571]: Wow, nice 🇩🇪

File: src/Chapter1.hs [hunk 3]
```diff
@@ -619,8 +631,13 @@ Implement a function that returns the sum of the last two digits of a number.
 Try to introduce variables in this task (either with let-in or where) to avoid
 specifying complex expressions.
 -}
+sumLast2 :: Int -> Int
+sumLast2 n = veryLast + secondToLast
+  where 
+    m = abs n
+    veryLast = lastDigit m
+    secondToLast = lastDigit (div m 10)
```
Comment (vrom911) [inline src/Chapter1.hs:639]: Nice reusage of functions 💯

**Review by vrom911 — APPROVED** [review]
Very accurate work! Congrats on finishing Chapter One 🎉 
I also learned a few German letters from your solutions, thanks! 😸 

Feel free to merge this PR as it is, it is great 💯

**Comment by wookenny** [comment]
Thanks you for your feedback :+1:

**Pull request merged**